recipe/meta.yaml

```diff
@@ -55,17 +56,16 @@ requirements:
     - re2
     - optional-lite
     - nsync
-    - libprotobuf
+    - libprotobuf 3.19.1
     # We need to statically link protobuf until we link against a system libonnx
     # See: https://github.com/conda-forge/onnxruntime-feedstock/issues/5
-    - libprotobuf-static
+    - libprotobuf-static 3.19.1
     - numpy
     - pybind11
     - safeint
   run:
     - coloredlogs
     - packaging
-    - protobuf
```

Comment:
This is the python package, not the C++ one. Is that really not used?

Reply:
Thanks, reverted.